examples/platform/silabs/SiWx917/SiWx917/hal/rsi_hal_mcu_interrupt.c

```diff
@@ -73,27 +39,11 @@ void rsi_gpio_irq_cb(uint8_t irqnum)
  */
 void rsi_hal_intr_config(void (*rsi_interrupt_handler)(void))
 {
-    call_back = rsi_interrupt_handler;
-    WFX_RSI_LOG("RSI:Set SPI intr CB to=%x", (uint32_t) call_back);
-}
 
-/*===================================================*/
-/**
- * @fn           void rsi_hal_log_stats_intr_config(void (* rsi_give_wakeup_indication)())
- * @brief        Checks the interrupt and map/set gpio callback function
- * @param[in]    rsi_give_wakeup_indication() ,gpio call back function to handle interrupt
- * @param[out]   none
- * @return       none
- * @description  This HAL API should contain the code
- *               related to mapping of gpio callback function.
- */
-#ifdef LOGGING_STATS
-void rsi_hal_log_stats_intr_config(void (*rsi_give_wakeup_indication)())
-{
-    gpio_callback = rsi_give_wakeup_indication;
-}
-#endif
+    // Configure interrupt pin/register in input mode and register the interrupt handler
```

Comment:
What happened with this file? It doesn't do anything anymore

Reply:
removed the file which are being used directly from the submodule
`  "${wiseconnect_sdk_root}/platforms/si91x/hal/src/rsi_bootup_config.c",
  "${wiseconnect_sdk_root}/platforms/si91x/hal/src/rsi_hal_mcu_m4.c",
  "${wiseconnect_sdk_root}/platforms/si91x/hal/src/rsi_hal_mcu_m4_rom.c",
  "${wiseconnect_sdk_root}/platforms/si91x/hal/src/rsi_hal_mcu_interrupt.c",`